packages/core/contracts/snapshot-helpers/SnapshotVotingPower.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.16;
+
+/**
+ * @title VotingInterface
+ * @notice Minimum interface required to interact with VotingV2 contract.
+ */
+interface VotingInterface {
+    struct VoterStake {
+        uint128 stake;
+        uint128 pendingUnstake;
+        uint128 rewardsPaidPerToken;
+        uint128 outstandingRewards;
+        int128 unappliedSlash;
+        uint64 nextIndexToProcess;
+        uint64 unstakeTime;
+        address delegate;
+    }
+
+    function voterStakes(address)
+        external
+        view
+        returns (
+            uint128,
+            uint128,
+            uint128,
+            uint128,
+            int128,
+            uint64,
+            uint64,
+            address
+        );
```

Comment:
I assume this is technically equivalent, but why not return memory VoterStake to be safe and access fields by name?

Reply:
ye, I think thats correct!

Reply:
agree, fixed in 6618296b3d3882e2ded245c27d6fa1b158b8f06a